### PR TITLE
[doc] add install_xtion2 doc

### PR DESCRIPTION
**Pull request by knorth55**
Add doc of `Xtion2` using by `openni2_camera`
cc. @YutoUchimi

### CODE REVIEW

File: doc/install_xtion2.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  ./SimpleViewer  # This should open a viewer for depth image
+
+
+Use Xtion2 camera with ``openni2_camera`` ROS package
+-----------------------------------------------------
+
+.. code-block:: bash
+
+  cd ~/ros/indigo/src
+  wstool set ros-drivers/openni2_camera https://github.com/ros-drivers/openni2_camera.git --git -vindigo-devel -y -u
```
Comment (k-okada) [inline doc/install_xtion2.rst]: `-vindigo-devel` typo?

Comment (k-okada) [reply doc/install_xtion2.rst]: please specify which version/commit we need, so we can decide when to use binary deb.

**Comment by furushchev** [comment]
@knorth55 Nice docs!

I'd like to report my situation:
In my laptop (ThinkPad T460s, Ubuntu 14.04.5), an error occurs when `SimpleViewer` is executed.

```bash
$ ./SimpleViewer 
Video Camera at /dev/video0 Detected
VIDIOC_STREAMON error 28, No space left on device
```

Reading a related post on ROS Answers:
https://answers.ros.org/question/12582/usb_cam-vidioc_streamon-error-28/
in this case, the bandwidth of the USB controller connected to the sensor should be wider, but there is the only one controller on my laptop, so I finally could not use this sensor.
Please add a note in such a case?
e.g. If you see `VIDIOC_STREAMON error 28, No space left on device`, try other ports first.

File: doc/install_xtion2.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+  # Download SDK.zip from https://www.asus.com/3D-Sensor/Xtion-2/HelpDesk_Download/  
+  cd ~/Downloads
+  unzip SDK.zip
+  cd SDK/Linux
+
+  tar zxvf ASUS-Linux-x64-OpenNI2.2.tar.gz
+  cd ASUS-Linux-x64-OpenNI2.2/
+
+  sudo ./install.sh
```
Comment (furushchev) [inline doc/install_xtion2.rst]: Before this line users must execute:

```bash
sudo apt install ros-indigo-openni2-launch ros-indigo-openni2-camera libopenni2-dev
```

File: doc/install_xtion2.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+.. code-block:: bash
+
+  cd ~/ros/indigo/src
+  wstool set ros-drivers/openni2_camera https://github.com/ros-drivers/openni2_camera.git --git -vindigo-devel -y -u
+
+  source /opt/ros/indigo/setup.bash
+  catkin b
+
+  source ~/ros/indigo/setup.bash
```
Comment (furushchev) [inline doc/install_xtion2.rst]: `~/ros/indigo/devel/setup.bash`?

File: doc/install_xtion2.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Use Xtion2 camera with ``openni2_camera`` ROS package
+-----------------------------------------------------
+
+.. code-block:: bash
+
+  cd ~/ros/indigo/src
+  wstool set ros-drivers/openni2_camera https://github.com/ros-drivers/openni2_camera.git --git -vindigo-devel -y -u
+
+  source /opt/ros/indigo/setup.bash
```
Comment (furushchev) [inline doc/install_xtion2.rst]: `~/ros/indigo/devel/setup.bash`?

**Comment by YutoUchimi** [comment]
```
$ ./SimpleViewer 
Video Camera at /dev/video0 Detected
VIDIOC_STREAMON error 28, No space left on device
```
I got this error, too. This was solved by
```
$ sudo rmmod uvcvideo
$ sudo modprobe uvcvideo quirks=128
```
But this will be reset every reboot (http://renoirsrants.blogspot.jp/2011/07/multiple-webcams-on-zoneminder.html), so I typed following command and then rebooted.
```
$ echo "options uvcvideo quirks=128" | sudo tee /etc/modprobe.d/uvcvideo.conf
```

**Comment by furushchev** [comment]
@YutoUchimi I run the command on my laptop, but still no luck. Maybe this depends on environment.

File: doc/install_xtion2.rst (hunk 4 above)
Comment (knorth55) [reply doc/install_xtion2.rst]: i think this is ok.

**Comment by knorth55** [comment]
Updated.
I have no idea why other computers have some errors on installation.
Oshiro and I can install the camera with this installation.

**Comment by YutoUchimi** [comment]
On my laptop (ThinkPad T460s, Ubuntu14.04 LTS) :
`lsusb` detects camera device, but when I run `./SimpleViewer`, device was not  found.

At that time Secure Boot on BIOS was enabled, so I tried disabling it, and after that I ran `sudo ./install.sh` as the installation guide shows.

Now I tried to launch camera, but the result was similar to that I saw on my desktop before.
```
$ ./SimpleViewer 
Video Camera at /dev/video1 Detected
Depth Camera at /dev/video0 Detected
After initialization:

Creating Depth stream
Depth Stream start
Creating color stream
VIDIOC_STREAMON error 28, No space left on device
```
This was not solved by the command I reported (same as @furushchev ?)

**Comment by YutoUchimi** [comment]
@Affonso-Gui and @itohdak tried to install Xtion2 just now.

- @Affonso-Gui 's case :
  - Environment : ThinkPad T440p, Ubuntu 14.04 LTS
  - Result : Successfully installed
  - When he ran `./SimpleViewer`, the following error occurs : `No space left on device`.
After `sudo rmmod uvcvideo` and `sudo modprobe uvcvideo quirks=128`, the viewer showed depth image properly.


- @itohdak  's case :
  - Environment : ThinkPad T460s, Ubuntu 14.04 LTS
  - Result : Failed to install
  - When he ran `./SimpleViewer`, the following error occurs : `No space left on device`.
After `sudo rmmod uvcvideo` and `sudo modprobe uvcvideo quirks=128`, the same error message appeared.

**Comment by knorth55** [comment]
Can you paste kernel version?

**Comment by YutoUchimi** [comment]
- @YutoUchimi 's desktop (Succeeded)
```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.5 LTS"
$ uname -a
Linux yuto-desktop 4.4.0-83-generic #106~14.04.1-Ubuntu SMP Mon Jun 26 18:10:19 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

```

- @YutoUchimi 's laptop (Failed)
```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.5 LTS"
$ uname -a
Linux yuto-ThinkPad-T460s 4.4.0-97-generic #120~14.04.1-Ubuntu SMP Wed Sep 20 15:53:13 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

```

**Comment by knorth55** [comment]
my desktop's kernel is as below;
i think that the kernel version is not a problem.
```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.5 LTS
$ uname -a
Linux geneve 4.4.0-96-generic #119~14.04.1-Ubuntu SMP Wed Sep 13 08:40:48 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

**Pull request merged**